README.md

```diff
@@ -26,13 +26,13 @@ parameter_posterior = infer(simulator, prior, method='SNPE', num_simulations=100
 
 ## Installation
 
-`sbi` requires Python 3.6 or higher. We recommend to use a [`conda`](https://docs.conda.io/en/latest/miniconda.html) virtual
+`sbi` requires Python 3.8 or higher. We recommend to use a [`conda`](https://docs.conda.io/en/latest/miniconda.html) virtual
 environment ([Miniconda installation instructions](https://docs.conda.io/en/latest/miniconda.html])). If `conda` is installed on the system, an environment for
 installing `sbi` can be created as follows:
 
 ```commandline
-# Create an environment for sbi (indicate Python 3.6 or higher); activate it
-$ conda create -n sbi_env python=3.7 && conda activate sbi_env
+# Create an environment for sbi (indicate Python 3.8 or higher); activate it
+$ conda create -n sbi_env python=3.8 && conda activate sbi_env
```

Comment:
change this to 3.10? because might be copying this snippet for their installation.

Reply:
Yes!

Reply:
we also need to fix the corresponding pre-commit version